rest/test/plugins/receipts/ReceiptsDb_spec.js

```diff
@@ -181,12 +185,12 @@ describe('receipts db', () => {
 				// Arrange:
 				const dbReceipts = [
 					createTransactionStatement(10, 100, [createReceipt({ recipientAddress: testAddress1 })]),
-					createTransactionStatement(20, 110, [
+					createTransactionStatement(20, 100, [
 						createReceipt({ recipientAddress: testAddress1 }),
 						createReceipt({ recipientAddress: testAddress2 })
 					]),
-					createTransactionStatement(30, 120, [createReceipt({ recipientAddress: testAddress2 })]),
-					createTransactionStatement(40, 130, [])
+					createTransactionStatement(30, 100, [createReceipt({ recipientAddress: testAddress2 })]),
+					createTransactionStatement(40, 100, [])
```

Comment:
why did you need to change these?

Reply:
I was testing the unique ids, it's tested elsewhere so rolling this back